.github/workflows/bump_version.yml

```diff
@@ -9,32 +11,8 @@ on:
 
 jobs:
```

Comment:
I feel this action can be merged into the version_upgrade.yml and both are just upgrading the package version

Reply:
if the `version_upgrade.yml` is used by another action then leave it.

Reply:
yup I refactored it to be reused.